src/main/java/br/com/alura/comex/modelo/PedidoEntity.java

```diff
@@ -0,0 +1,79 @@
+package br.com.alura.comex.modelo;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import br.com.alura.comex.enums.TipoDescontoEnum;
+
+@Entity
+@Table(name = "Pedido")
+public class PedidoEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	private LocalDate data = LocalDate.now();
+	private double desconto;
+
+	@Enumerated(EnumType.STRING)
+	private TipoDescontoEnum tipo_desconto;
+
+	@OneToMany(mappedBy = "pedido_id", cascade = CascadeType.ALL)
+	private List<ItemDePedidoEntity> item_pedido = new ArrayList<>();
+
+	@ManyToOne
+	// @Column(name = "id_cliente")
+	private ClienteEntity id_cliente;
```

Comment:
Para definir o nome da coluna, se for diferente do padrão, tem que usar `@JoinColumn`